Misc/NEWS.d/next/Build/2022-02-04-21-26-50.bpo-46640.HXUmQp.rst

```diff
@@ -0,0 +1,2 @@
+Building Python now requires a C99 ``<math.h>`` header file providing the
```

Comment:
Why are there two separate `Misc/NEWS.d` files for this PR?

Reply:
Ah, I see. One's for the build changes, one for the C API changes. Got it, but the wording here needs updating, since it's only true that building requires the `NAN` constant on a platform that doesn't define `Py_NO_NAN`.

Reply:
I removed it.